tests/filecheck/backend/csl/print_csl.mlir

```diff
@@ -780,6 +783,12 @@ csl.func @builtins() {
 // CHECK-NEXT:     .wavelet_index_offset = false,
 // CHECK-NEXT:     .control = true,
 // CHECK-NEXT:   }});
+// CHECK-NEXT:   const zero_stride_dsd : mem4d_dsd = @get_dsd( mem4d_dsd, .{
+// CHECK-NEXT:     .tensor_access = | d0, d1, d2 | { i16_value, i16_value, i16_value } -> A[ d2 ]
+// CHECK-NEXT:   });
+// CHECK-NEXT:   const one_zero_stride_dsd : mem4d_dsd = @get_dsd( mem4d_dsd, .{
+// CHECK-NEXT:     .tensor_access = | d0, d1, d2 | { i16_value, i16_value, i16_value } -> A[ 0 * d0, d1, d2 ]
```

Comment:
This would not work. `A` is a 1D array being accessed as a 3D array. I think this is an error elsewhere in the printer.

Reply:
Actually, the first one should work. The second one I added only to check that the throw-away-index trick is only used when all except one strides are 0.

Reply:
I took it out.

Reply:
I tried compiling it, and it doesn't compile...

```zig

var A : [24]f32 = @constants([24]f32, 0.0);
const one_zero_stride_dsd : mem4d_dsd = @get_dsd( mem4d_dsd, .{
  .tensor_access = | d0, d1, d2 | { i16_value, i16_value, i16_value } -> A[ 0 * d0, d1, d2 ]
});
```

Gives the error

```
./pe_program.csl:98:74: error: incompatible tensor with rank 1
  .tensor_access = | d0, d1, d2 | { i16_value, i16_value, i16_value } -> A[ 0 * d0, d1, d2 ]
                                                                         ^
./pe_program.csl:98:20: error: in tensor-access with 3 access expression(s) provided
  .tensor_access = | d0, d1, d2 | { i16_value, i16_value, i16_value } -> A[ 0 * d0, d1, d2 ]
                   ^
./pe_program.csl:96:1: error: declared here
var A : [24]f32 = @constants([24]f32, 0.0);
^
```